visualization/app/codeCharta/ui/codeMap/codeMap.label.service.ts

```diff
@@ -118,7 +118,10 @@ export class CodeMapLabelService implements CameraChangeSubscriber {
 		const multiLineContext = message.split("\n")
 
 		// setting canvas width/height before ctx draw, else canvas is empty
-		canvas.width = context.measureText(message).width + margin
+		canvas.width =
+			context.measureText(multiLineContext[0]).width > context.measureText(multiLineContext[1]).width
+				? context.measureText(multiLineContext[0]).width + margin
+				: context.measureText(multiLineContext[1]).width + margin
```

Comment:
```suggestion
		const firstMultiLineContextWidth = context.measureText(multiLineContext[0]).width
		const secondMultiLineContextWidth = context.measureText(multiLineContext[1]).width
		canvas.width =
			firstMultiLineContextWidth > secondMultiLineContextWidth
				? firstMultiLineContextWidth + margin
				: secondMultiLineContextWidth + margin
```

Reply:
That is more performant, thanks for the feedback